subjects/DPxAI/robots-harmony/README.md

```diff
@@ -0,0 +1,394 @@
+
+#### 2- Create a Container for the Robots:
+
+Inside the `<body>`, create a div element with the ID gallery.
+
+This div will serve as the container for all the robot portraits.
+
+```html
+<div id="gallery"></div>
+```
+
+#### 3- Include a title and each Robot:
+
+- Inside your gallery `<div>`, create an `<h1>` tag inside a `<div>` with an `id` of `title`. Then put the name of your gallery inside it !
+
+```HTML
+    <div id="title">
+        <h1>your favorite gallery name</h1>
+    </div>
+```
+
+- Each team member should copy their robot's HTML structure (the one you provided in the `first-move.html` exercise) and paste it inside the `gallery` div.
+
+- Ensure each robot is placed inside its own div with the class `name-portrait`, for example:
+
+```html
+<div class="john-robot">
+  <!-- Paste your robot's HTML structure here -->
+</div>
+<div class="sarah-robot">
+  <!-- Paste your robot's HTML structure here -->
+</div>
+<div class="bob-robot">
+  <!-- Paste your robot's HTML structure here -->
+</div>
+```
+
+#### 4- Add Robot Information:
+
+Under each robot's `</section>`, add an `<h3>` element for the robot's name and a `<p>` element for a short description of its power.
+
+```html
+<h3>Your robot's name</h3>
+<p>Your robot's description</p>
+```
+
+### Task 2: Style the Gallery
+
+#### 1- Update the CSS File:
+
+- In your `robots-harmony.css` file, add these styles:
+
+```css
+* {
+  margin: 0;
+  box-sizing: border-box;
+  opacity: 0.85;
+}
+
+body {
+  height: 100vh;
+}
+
+section {
+  padding: 20px;
+  width: 100%;
+  height: auto;
+  display: flex;
+  justify-content: center;
+}
+
+div,
+p {
+  border: solid 1px black;
+  padding: 10px;
+  margin: 0;
+  border-radius: 30px;
+}
+
+#face {
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+#eyes {
+  display: flex;
+  background-color: yellow;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: 50px;
+  width: 200px;
+}
+
+#torso {
+  width: 150px;
+  height: 90%;
+  background-color: violet;
+}
+
+.eye {
+  width: 60px;
+  height: 60px;
+  background-color: red;
+  border-radius: 50%;
+}
+
+.arm {
+  background-color: aquamarine;
+  width: 50px;
+  height: 100px;
+  margin: 20px;
+}
+
+.leg {
+  background-color: dodgerblue;
+  width: 50px;
+  height: 100px;
+  margin: 20px;
+}
+
+.body-member {
+  width: 50px;
+  margin: 30px;
+}
+
+h3 {
+  margin-bottom: 10px;
+}
+
+#title {
+  text-align: center;
+  margin: 1rem;
+  background-color: #383086;
+  color: rgb(158, 235, 223);
+}
+```
+
+- Then add this gallery style block, and change the `background-color` of it based on your team's favorite color:
+
+```css
+#gallery {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 100px;
+  padding: 100px;
+  /* Change this color to reflect the color of your gallery */
+  background-color: #383086;
+}
+```
+
+- Add the following block of style to all you class's of `name-robot`. To do so, we follow the rule:
```

Comment:
typo here in the word "all 'you' class" ?

Reply:
Fixed.